packages/react-tinacms-strapi/src/strapi-media.store.ts

```diff
@@ -0,0 +1,54 @@
+import { Media, MediaUploadOptions } from '@tinacms/media'
+
+import Cookies from 'js-cookie'
+import { STRAPI_JWT } from './tina-strapi-client'
+
+export class StrapiMediaStore {
+  accept = '*'
+
+  async persist(files: MediaUploadOptions[]): Promise<Media[]> {
+    const uploaded: Media[] = []
+
+    for (const { file } of files) {
+      const upload = await uploadFile(file)
+      uploaded.push({
+        directory: '/uploads',
+        filename: upload[0].hash + upload[0].ext + `?${upload[0].id}`,
+      })
+    }
+    return uploaded
+  }
+
+  getFilePath(fileUrl: string): string {
+    return fileUrl.split('?')[0]
+  }
+
+  getFileId(fileUrl: string): string {
+    return fileUrl.split('?')[1]
+  }
+
+  getAbsolutePath(fileUrl: string): string {
+    return process.env.STRAPI_URL + fileUrl
+  }
+}
+
+export async function uploadFile(file: File) {
+  const authToken = Cookies.get(STRAPI_JWT)
+  const formData = new FormData()
+  formData.append('files', file)
+  const uploadResponse: Response = await fetch(
+    process.env.STRAPI_URL + '/upload',
```

Comment:
Is there a reason why this function is not a method on `StrapiMediaStore`? 

I would lean towards 
1. Make it a method
1. Remove the `process.env.STRAPI_URL` access in favor of an instance variable set on media store at initialization.